SpatialGDK/Extras/insights-proxy/README.md

```diff
@@ -0,0 +1,2 @@
+# Insights Proxy
+This Python script is used to allow proxying insights data from a SpatialOS worker to Insights running on the local machine. See UNR-5207 for more information.
```

Comment:
So is this directly invoked on a dev machine? This is python 2.x code at the moment, would be good to describe this dependency, would also prefer to be python 3.6+ based personally

Reply:
Yeah, and run manually. Good point about Python versions, is it possible to make code compatible with as many (all?) versions as possible?

Reply:
3.6 allows for format string syntax. But I don't think I got who the target audience was until I dived into the docs just now. If you want wide support use `print()` other than that your code should work in python 2 & 3. If you want to add the string formatting suggestion I commented above you'll need to restrict yourself to c style string formatting: e.g. `print('Connecting to %s:%d [Insights]' % (ip, port))`

Reply:
It's worth noting that unreal comes packaged with python 3.7 (turned on by default) and 2.7 (turned off by default)

Reply:
I think the workflow doc describes a workflow that depends on whatever python you have installed on PATH

Reply:
So actually you could run this from unreal directly? https://docs.unrealengine.com/en-US/ProductionPipelines/ScriptingAndAutomation/Python/index.html